rllib/algorithms/dqn/dqn.py

```diff
@@ -655,6 +655,7 @@ def _training_step_new_api_stack(self):
                         batch_length_T=self.env_runner.module.is_stateful()
                         * self.config.model_config.get("max_seq_len", 0),
                         lookback=int(self.env_runner.module.is_stateful()),
+                        burnin=self.config.burnin,
```

Comment:
so I think with the suggestions above, we won't even need to change the buffer anymore, correct?